src/components/Home/__tests__/Home.test.tsx

```diff
@@ -0,0 +1,90 @@
+import { cleanup, waitFor } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import { Home } from "../Home";
+import { render } from "../../../../testing";
+
+const events = [{
+  "id": "tb34kdqm2cr96rvk2blhprpgks_20230503T083000Z",
```

Comment:
uhh dunno if we want the link for our standups here ahah, just add other characters or something

Reply:
done